packages/atomic/src/components/common/facets/facet-container/facet-container.spec.ts

```diff
@@ -0,0 +1,32 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {html, TemplateResult} from 'lit';
+import {expect, describe, it} from 'vitest';
+import {renderFacetContainer} from './facet-container';
+
+describe('renderFacetContainer', () => {
+  const setupElement = (children: TemplateResult) => {
+    return renderFunctionFixture(html`${renderFacetContainer()(children)}`);
+  };
+
+  const getContainer = (element: HTMLElement) =>
```

Comment:
Maybe refactor with a utility:
```ts
const getPartElement = (element: HTMLElement, part: string) => element.querySelector(`*[part="${part}"]`);
```

Reply:
It can create confusion because this will not work for element with more than one part